src/Features/Onboarding.php

```diff
@@ -1053,8 +1053,17 @@ public static function reset_task_list() {
 			return;
 		}
 
-		$new_value = 1 === absint( $_GET['reset_task_list'] ) ? 'no' : 'yes'; // phpcs:ignore CSRF ok.
+		$previous_value = 1 === absint( $_GET['reset_task_list'] ) ? 'yes' : 'no'; // phpcs:ignore CSRF ok.
+		$new_value      = 'no' === $previous_value ? 'yes' : 'no'; // phpcs:ignore CSRF ok.
+
 		update_option( 'woocommerce_task_list_hidden', $new_value );
+		wc_admin_record_tracks_event(
+			'wcadmin_tasklist_toggled',
```

Comment:
You'll get a double prefixed event name, `tasklist_toggled` will suffice.

https://github.com/woocommerce/woocommerce/blob/251e1ea27305234220b9b7952b08cd7e0174adbf/includes/tracks/class-wc-tracks.php#L89

Reply:
Awesome thanks for spotting this. will fix.